fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -995,7 +996,7 @@ public int getExecTimeout() {
             // particular for insert stmt, we can expand other type of timeout in the same way
             return Math.max(sessionVariable.getInsertTimeoutS(), sessionVariable.getQueryTimeoutS());
         } else if (executor != null && executor.isAnalyzeStmt()) {
-            return sessionVariable.getAnalyzeTimeoutS();
+            return StatisticConstants.ANALYZE_TIMEOUT_IN_SEC;
```

Comment:
For `analyze` stmt, why use fixed timeout value?

Reply:
AnalyzeStmt is the statement executed by mysql connection thread. Before, AnalyzeStmt was using analyze_timeout, but it brings the problem that AnalyzeStmt may be timeout but the internal analyze sql has not cancelled yet, in this case, the cancellation may fail. So here use a fixed timeout value for AnalyzeStmt and use analyze_timeout to control the analyze sql timeout. When analyze sql timeout, AnalyzeStmt will get an exception and return.